### PR TITLE
Oil Gas Wells

### PR DESCRIPTION
**Pull request by miriamseely**
Current draft for the metadata for the [Utah Oil Gas Wells](https://gis.utah.gov/products/sgid/energy/oil-gas-wells/) layer. I had a few questions about this one, which I left as comments on the md file. Let me know what you think! This PR also includes a separate document for the field definitions for this layer.

### CODE REVIEW

File: metadata/energy/oil_gas_wells.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+### What is the dataset?
+
+Oil and gas wells in Utah are managed by the Utah Department of Natural Resources [Oil, Gas and Mining Division](https://ogm.utah.gov/). This dataset contains all oil and gas wells in Utah, including those being actively drilled, still in the permit process, and those that have been abandoned.
+
+### What is the purpose of the dataset?
+
+This dataset provides the opportunity for the Division to collect more descriptive down-hole information for each wellbore permitted and drilled.
+
+<!--- This is from the original metadata. Do we still see this dataset as an opportunity to collect more descriptive information, or does this dataset have a different purpose now? --->
```
Comment (stdavis) [inline metadata/energy/oil_gas_wells.md:27]: I'm not sure where this purpose came from. I would say that it's purpose is to show the surface locations for wells. We do not participate in any data collection for this dataset.

File: metadata/energy/oil_gas_wells.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+### What does the dataset represent?
+
+Points in this dataset include the API code, well and company name, account number, filed number, field name, elevation, locations coordinates, lease numbers, well type and status, total cumulative oil, gas and water, and many other attributes.
+
+### How was the dataset created?
+
+This data was developed by contractors sponsored in part by the U.S. Department of Energy and the Ground Water Protection Council.
+
+<!--- What else might the user need to know about how this dataset was created? Do we maintain the dataset now, or is maintained by someone else? --->
```
Comment (stdavis) [inline metadata/energy/oil_gas_wells.md:37]: It's maintained by DOGM. We pull the data via an automated process. There's not much translation other than converting coordinate fields to shapes.

File: metadata/energy/oil_gas_wells.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+### How reliable and accurate is the dataset?
+
+Initially, much of the down-hole information will not be available because there was no place to save it in the previous databases. Eventually, as new wells are drilled and old legacy data is entered, more descriptive data will become available for the wells in Utah.
```
Comment (stdavis) [inline metadata/energy/oil_gas_wells.md:41]: This is incorrect. Down-hole information is a [separate dataset](https://opendata.gis.utah.gov/datasets/utahDNR::oil-and-gas-down-hole-locations/about).

Yes, this is the most complete and current dataset for wells. It's actively maintained by DOGM.

**Review by stdavis — COMMENTED** [review]
This is looking like a good start. Once we've done everything that we can do, I'd like to send this to DOGM for their review.

Note that there are two other related datasets:
[Oil Gas Wells Down Holes](https://opendata.gis.utah.gov/datasets/utahDNR::oil-and-gas-down-hole-locations/about)
[Oil Gas Wells Paths](https://opendata.gis.utah.gov/datasets/utahDNR::oil-and-gas-wells/about)

These do not have dedicated product pages on our website.

**Comment by miriamseely** [comment]
> Note that there are two other related datasets: [Oil Gas Wells Down Holes](https://opendata.gis.utah.gov/datasets/utahDNR::oil-and-gas-down-hole-locations/about) [Oil Gas Wells Paths](https://opendata.gis.utah.gov/datasets/utahDNR::oil-and-gas-wells/about)
> 
> These do not have dedicated product pages on our website.

I've got those ones on my list! Are there some resources I can use to find information about these layers? There's very little available on AGOL or the SGID index. @stdavis

File: metadata/energy/oil_gas_wells.md (hunk 2 above)
Comment (miriamseely) [reply metadata/energy/oil_gas_wells.md:37]: I'll be sure to add that in!

**Comment by steveoh** [comment]
> Are there some resources I can use to find information about these layers? 

You could see if there is anything useful on https://oilgas.ogm.utah.gov/oilgasweb/live-data-search/lds-main.xhtml or somewhere else on https://ogm.utah.gov/og-home/

**Review by stdavis — APPROVED** [review]

**Comment by stdavis** [comment]
This looks good to me. I sent an email to Aaron Brown and Michael Harte (both from DOGM) for their feedback on this.

**Comment by stdavis** [comment]
This is going to be very valuable. Thanks, @miriamseely!

**Pull request merged**